Index: selfoss-psc/selfoss/helpers/ContentLoader.php
===================================================================
--- selfoss-psc.orig/selfoss/helpers/ContentLoader.php
+++ selfoss-psc/selfoss/helpers/ContentLoader.php
@@ -263,7 +263,7 @@ class ContentLoader {
                 'keep_bad' => 0,
                 'comment' => 1,
                 'cdata' => 1,
-                'elements' => 'div,p,ul,li,a,img,dl,dt,dd,h1,h2,h3,h4,h5,h6,ol,br,table,tr,td,blockquote,pre,ins,del,th,thead,tbody,b,i,strong,em,tt,sub,sup,s,strike,code'
+                'elements' => 'div,p,ul,li,a,img,dl,dt,dd,h1,h2,h3,h4,h5,h6,ol,br,table,tr,td,blockquote,pre,ins,del,th,thead,tbody,b,i,strong,em,tt,sub,sup,s,strike,code,cite'
             ],
             'img=width, height'
         );
