Index: selfoss-psc/selfoss/public/js/selfoss-shortcuts.js
===================================================================
--- selfoss-psc.orig/selfoss/public/js/selfoss-shortcuts.js
+++ selfoss-psc/selfoss/public/js/selfoss-shortcuts.js
@@ -16,17 +16,6 @@ selfoss.shortcuts = {
             return false;
         });
 
-        // 'right cursor': next article
-        $(document).bind('keydown', 'right', function(e) {
-            if (!selfoss.shortcuts.active()) {
-                return false;
-            }
-
-            selfoss.shortcuts.entrynav('next');
-            e.preventDefault();
-            return false;
-        });
-
         // 'j': next article
         $(document).bind('keydown', 'j', function(e) {
             if (!selfoss.shortcuts.active()) {
@@ -59,17 +48,6 @@ selfoss.shortcuts = {
             e.preventDefault();
             return false;
         });
-
-        // 'left': previous article
-        $(document).bind('keydown', 'left', function(e) {
-            if (!selfoss.shortcuts.active()) {
-                return false;
-            }
-
-            selfoss.shortcuts.entrynav('prev');
-            e.preventDefault();
-            return false;
-        });
 
         // 'k': previous article
         $(document).bind('keydown', 'k', function(e) {
