Index: selfoss-psc/selfoss/public/js/selfoss-shortcuts.js
===================================================================
--- selfoss-psc.orig/selfoss/public/js/selfoss-shortcuts.js
+++ selfoss-psc/selfoss/public/js/selfoss-shortcuts.js
@@ -82,6 +82,18 @@ selfoss.shortcuts = {
             return false;
         });
 
+        // 'l': load images
+        $(document).bind('keydown', 'l', function(e) {
+            if (!selfoss.shortcuts.active()) {
+                return false;
+            }
+
+            selfoss.events.entriesToolbar($('.entry.selected'));
+            $('.entry.selected .entry-loadimages').click();
+            e.preventDefault();
+            return false;
+        });
+
         // 's': star/unstar
         $(document).bind('keydown', 's', function(e) {
             if (!selfoss.shortcuts.active()) {
