Index: selfoss-psc/selfoss/defaults.ini
===================================================================
--- selfoss-psc.orig/selfoss/defaults.ini
+++ selfoss-psc/selfoss/defaults.ini
@@ -17,8 +17,8 @@ username=
 password=
 salt=lkjl1289
 public=
-html_title=selfoss
-rss_title=selfoss feed
+html_title=Selfoss Feed Reader
+rss_title=Selfoss Feed Reader feed
 rss_max_items=300
 rss_mark_as_read=0
 homepage=newest
