Index: selfoss-psc/selfoss/public/js/selfoss-events-entries.js
===================================================================
--- selfoss-psc.orig/selfoss/public/js/selfoss-events-entries.js
+++ selfoss-psc/selfoss/public/js/selfoss-events-entries.js
@@ -6,7 +6,8 @@ selfoss.events.entries = function(e) {
     $('.entry, .entry-title').unbind('click');
 
     // show/hide entry
-    var target = selfoss.isMobile() ? '.entry' : '.entry-title';
+    //var target = selfoss.isMobile() ? '.entry' : '.entry-title';
+    var target = '.entry-title';
     $(target).click(function() {
         var parent = ((target == '.entry') ? $(this) : $(this).parent());
         
